### PR TITLE
Cleaned up unneeded appraisal subs

### PR DESCRIPTION
**Pull request by Nituki**
Since a few of these were added into toggles, I removed them to clean up the regex.

### CODE REVIEW

**Comment by rpherbig** [comment]
@KatoakDR @jayRyan24 @jandersson can one of you take a look at this?

**Comment by Hiinky** [comment]
I believe all of these have been recently moved to native Simu toggles.  mana levels, gear apps, and critter apps are what I'm remembering.  They also re-did critter appraisals a week or two ago so our current subs are not even accurate now.

The only one I would say keep is the one for encumbrance.  I don't see that simu one active.  Luck is but encumbrance is not.

I'm sending out an announcement in lich discord to remind people to turn on the simu toggles because we're turning these off.

```
>;k text  ##<<<<< turned off lich ;textsubs to test.
>perc 
>
You reach out with your senses and hear lambent (12/21) streams of harmonious Elemental mana coursing through the area.
Concentrating harder on the sounds you hear, you can sense luminous (14/21) mana to the southeast.
You sense the Rage of the Clans spell upon you, which will last for about seven roisaen.
You sense the Echoes of Aether spell upon you, which will last for about eight roisaen.
Roundtime: 3 sec.
>app odaj
The battle odaj is light armor.

The odaj looks like it offers protection for the following areas:

    right leg
    left leg
    chest
    abdomen
    back

You feel certain that an austere battle odaj of somber black leather appears to impose moderate (7/15) maneuvering hindrance and light (3/15) stealth hindrance, offering:
    great (9/15) protection and very good (8/17) damage absorption for puncture attacks.
    high (7/15) protection and very good (8/17) damage absorption for slice attacks.
    very high (8/15) protection and very good (8/17) damage absorption for impact attacks.
    good (5/15) protection and fair (5/17) damage absorption for fire attacks.
    good (5/15) protection and fair (5/17) damage absorption for cold attacks.
    moderate (4/15) protection and somewhat fair (4/17) damage absorption for electrical attacks.

If you were only wearing an austere battle odaj of somber black leather, your maneuvering would be minimally hindered (2/14), and your stealth would be minimally hindered (2/14).
But considering all the armor and shields you are wearing or carrying, you are currently fairly hindered (5/14), and your stealth is lightly hindered (4/14).

You are certain that the battle odaj is extremely resistant to damage (15/18) and is in pristine condition (100%).

The battle odaj is moderately soft.
You are certain that the battle odaj weighs exactly 160 stones.
You are certain that the battle odaj is worth exactly 270600 Dokoras.
Roundtime: 8 seconds.

  Encumbrance : Somewhat Burdened
         Luck : Average (0/3)
```

File: textsubs.lic [hunk 1]
```diff
@@ -784,19 +386,6 @@ TextSubs.add('decent (quantity|number) remains to be found', 'decent \1 (2/5) re
...
-TextSubs.add('Encumbrance : Somewhat Burdened', 'Encumbrance : Somewhat Burdened (2/11)')
-TextSubs.add('Encumbrance : Burdened', 'Encumbrance : Burdened (3/11)')
-TextSubs.add('Encumbrance : Heavy Burden', 'Encumbrance : Heavy Burden (4/11)')
-TextSubs.add('Encumbrance : Very Heavy Burden', 'Encumbrance : Very Heavy Burden (5/11)')
-TextSubs.add('Encumbrance : Overburdened', 'Encumbrance : Overburdened (6/11)')
-TextSubs.add('Encumbrance : Very Overburdened', 'Encumbrance : Very Overburdened (7/11)')
-TextSubs.add('Encumbrance : Extremely Overburdened', 'Encumbrance : Extremely Overburdened (8/11)')
-TextSubs.add('Encumbrance : Tottering Under Burden', 'Encumbrance : Tottering Under Burden (9/11)')
-TextSubs.add('Encumbrance : Are you even able to move?', 'Encumbrance : Are you even able to move? (10/11)')
-TextSubs.add('Encumbrance : It\'s amazing you aren\'t squashed!', 'Encumbrance : It\'s amazing you aren\'t squashed! (11/11)')
```
Comment (Hiinky) [inline textsubs.lic:798]: Leave these unless I missed another Simu toggle you can remind me about Nituki. 😁

Comment (Nituki) [reply textsubs.lic:798]: Nope you are right. I thought these were part of the luck update but had another sub running when I checked. These should be kept.

**Comment by rpherbig** [comment]
✔️ Changes look good. Add the encumbrance subs back in and I'll merge this

**Pull request merged**